migrations.js

```diff
@@ -79,15 +121,18 @@ var runMigrations = function(meta, migrations, callback) {
 
 module.exports = {
   run: function(callback) {
-    getMeta(function(err, meta) {
+    module.exports.get(function(err, migrations) {
```

Comment:
Appreciate this is not part of this change, but is `get()` only exported for unit tests?